crates/forge/src/pretty_printing.rs

```diff
@@ -72,7 +79,34 @@ pub(crate) fn print_test_result(test_result: &TestCaseSummary, fuzzer_runs: Opti
         }
     };
 
-    println!("{result_header} {result_name}{fuzzer_report}{result_message}");
+    println!("{result_header} {result_name}{fuzzer_report}{block_message}{result_message}");
+    Ok(())
+}
+
+fn get_block_message(fork_params: Option<&(Url, BlockId)>) -> Result<String> {
+    let result = if let Some((url, block_id)) = fork_params {
+        match block_id {
+            BlockId::Tag(_) => {
+                let client = JsonRpcClient::new(HttpTransport::new(url.clone()));
+                let runtime = Runtime::new().expect("Could not instantiate Runtime");
+                match runtime.block_on(client.get_block_with_txs(block_id))? {
+                    Block(BlockWithTxs {
+                        block_number,
+                        block_hash,
+                        ..
+                    }) => format!("\nBlock number: {block_number}, block hash: {block_hash}"),
+                    PendingBlock(PendingBlockWithTxs { parent_hash, .. }) => {
+                        format!("\nBlock parent hash: {parent_hash}")
+                    }
+                }
+            }
+            _ => String::new(),
+        }
+    } else {
+        String::new()
+    };
+
+    Ok(result)
```

Comment:
This is not a way to go. We shouldn't be spawning any runtimes in the code that just does printing. If it's impossible to get the actual block id used, code of cheatnet should be changed so this value is simply returned/accessible to us.

Reply:
There is an issue for that, it is a temporary solution #791